lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -207,32 +254,32 @@ def update_config(self, config_dict: Dict) -> None:
                     relation.data[self.model.app][key] = str(self.config_dict[key])
```

Comment:
Is the value in the dict always a string? If so, can you add the types in the method signature and remove the cast?

Reply:
IIRC there can be integers as well, so I'll keep the cast for now